apps/editing-toolkit/package.json

```diff
@@ -16,9 +16,9 @@
 	},
 	"homepage": "https://github.com/Automattic/wp-calypso",
 	"scripts": {
-		"full-site-editing": "calypso-build --source='dotcom-fse'",
-		"dev:full-site-editing": "yarn run full-site-editing",
-		"build:full-site-editing": "NODE_ENV=production yarn run full-site-editing",
+		"dotcom-fse": "calypso-build --source='dotcom-fse'",
```

Comment:
At times I still confuse myself because there are two dotcom fse builds. 

1. The team's initial attempt at dotcom fse in the editing toolkit
2. The core implementation of dotcom fse in gutenberg

This is `non-blocking`, and it doesn't have to happen in this PR, but I'm curious to hear what you think @noahtallen. I wonder if we could rename this to something that indicates that we _are not_ building on top of this dotcom-fse implementation anymore. `dotcom-fse-deprecated` `dotcom-fse-v0` etc.

Reply:
Thanks for mentioning that! So, from my point of view, it works because:

 > The core implementation of dotcom fse in gutenberg

we aren't implementing "dotcom" fse in Gutenberg. Like, the "dotcom" part is supposed to indicate that this is the "WordPress.com" solution we tried to implement, as opposed to any core-first implementation which came after.

Adding deprecated certainly wouldn't hurt, i think that would be a decent addition to clarify even further.